website/docs/for-ios/how-to/sharing-assets.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Sharing Assets Between Portals
+sidebar_label: Sharing Assets
+---
+
+# Overview
+
+If you are devloping an application that contains multiple Portals, the Portals library supports the ability to share asset files between them. This is helpful to reduce the size of your overall app if those Portals use assets that are the same such as large media files or font files.
+
+## Register Shared Assets
+
+Shared Assets should be placed inside both native platforms in an accessible location and registered in the Portals Library. This provides the Web content with a mechanism to get a path to the shared assets within the native platform bundle.
+
+### iOS
```

Comment:
This heading isn't really needed since it's under the iOS docs

Reply:
Thanks, cleaned them both up now @Steven0351